tutorial/part-1-creating-a-simple-view/README.md

```diff
@@ -1,18 +1,17 @@
 # Part 1: Creating a simple view
 
-Let’s learn by building a small app.  
+Let's build your first app.
 
-
-Through this tutorial, we’ll walk you through the creation of a basic Grocery Catalog application for an online grocery store called Oakry. The application is a multi-page dashboard used by Oakry’s Catalogue Managers to:
+In this tutorial, we’ll create a Grocery Catalog application for an online grocery store called "Oakry".  Oakry’s Catalogue Managers can use the multi-page application to:
 
 * View all products
 * Add a new product
-* Edit a product’s details
+* Edit details of a product
 
-We recommend that you use [Appsmith on Cloud](https://docs.appsmith.com/quick-start#appsmith-cloud) for this tutorial. This is the quickest way to get started. Alternatively, you can also [set up Appsmith locally, or on your private instance](https://docs.appsmith.com/quick-start#docker). We’ll assume that you have a working Appsmith deployment before you start the tutorial.   
+We recommend that you use [Appsmith on Cloud](https://docs.appsmith.com/quick-start#appsmith-cloud) for this tutorial. This is the quickest way to get started. Alternatively, you can also [set up Appsmith locally, or on your private instance](https://docs.appsmith.com/quick-start#docker).   
```

Comment:
It's typically good to explicitly mention this so that they don't try to attempt anything and raise those as errors or queries on support. Can you please elaborate on why you think this is unnecessary?

Reply:
These feel like unnecessary words because we are explaining something obvious. Readers will know that they can't do the tutorial if they don't have a running deployment. ANd people who have errors should still raise those as support tickets.